addon/mixins/data-adapter-mixin.js

```diff
@@ -20,7 +19,15 @@ import { isPresent } from '@ember/utils';
   import DataAdapterMixin from 'ember-simple-auth/mixins/data-adapter-mixin';
 
   export default DS.JSONAPIAdapter.extend(DataAdapterMixin, {
-    authorizer: 'authorizer:application'
+      headers: computed('session.data.authenticated.token', function() {
+        let headers = {};
+        if (this.session.isAuthenticated) {
+          headers['Authorization'] = `Bearer ${this.session.data.authenticated.token}`;
+        }
+
+        return headers;
+      }),
+    }
```

Comment:
The indentation seems off, and the last `}` is not needed, the line below is closing the block.

Reply:
fixed 👍